openshift/e2e-tests-openshift.sh

```diff
@@ -53,33 +54,88 @@ function install_strimzi(){
   sleep 5; while echo && kubectl get pods -n kafka | grep -v -E "(Running|Completed|STATUS)"; do sleep 5; done
 }
 
+function install_servicemesh(){
+  header "Installing ServiceMesh"
+
+  # Install the ServiceMesh Operator
+  oc apply -f https://raw.githubusercontent.com/openshift/knative-serving/release-v0.8.1/openshift/servicemesh/operator-install.yaml
+
+  # Wait for the istio-operator pod to appear
+  timeout_non_zero 900 '[[ $(oc get pods -n openshift-operators | grep -c istio-operator) -eq 0 ]]' || return 1
+
+  # Wait until the Operator pod is up and running
+  wait_until_pods_running openshift-operators || return 1
+
+  # Deploy ServiceMesh
+  oc new-project $SERVICEMESH_NAMESPACE
+  oc apply -n $SERVICEMESH_NAMESPACE -f https://raw.githubusercontent.com/openshift/knative-serving/release-v0.8.1/openshift/servicemesh/controlplane-install.yaml
+  cat <<EOF >> ServiceMeshMemberRoll.yaml
```

Comment:
nit: It'd be nicer to ditch the ServiceMeshMemberRoll.yaml file and just feed the content into oc

Reply:
How 'bout this for an idea:
```
testNames=$(cat TEST_NAMES | tr "\n" " ")

cat <<EOF | oc apply -f -
apiVersion: maistra.io/v1
kind: ServiceMeshMemberRoll
metadata:
  name: default
  namespace: ${SERVICEMESH_NAMESPACE}
spec:
  members:
  - ${SERVING_NAMESPACE}
  - ${EVENTING_NAMESPACE}
$(for i in $testNames; do echo "  - $i"; done)
EOF
)
```